indigo_app/templates/place/works.html

```diff
@@ -22,7 +22,11 @@
     </div>
   {% endblock %}
 
-  <div class="card">
+<div class="row">
+  <div data-vue-component="TaxonomyTOC"></div>
+  {{ taxonomy_toc|json_script:"taxonomy_toc" }}
+
+  <div class="card col-md mx-1 p-0">
```

Comment:
wrap the `.card` in a `.col`, you can't combine them. Don't mess with the `.col`'s padding or margin.

Reply:
changed :white_check_mark: